COOKBOOK.md

```diff
@@ -0,0 +1,81 @@
+# Guideline for writing Dojo 2 cookbook recipes
+
+## Purpose
+
+This guideline describes how to create a Dojo 2 cookbook recipe.
+
+Recipes are intended to help consumers find easily consumable solutions to challenges that they are addressing in their projects. Recipes are focused on providing clear, concise steps showing how to accomplish a specific, narrowly defined task. They are almost completely focused on _what_ to do instead of _why_ to do something. If additional explanation is required, links should be provided to relevant documentation for consumers that wish to learn more about a specific topic.
+
+## Summary
+
+The following rules should be followed when creating a cookbook recipe. See the [discussion](#discussion) section for more information about each point.
+
+- [ ] [Contains proper front matter](#contains-proper-front-matter)
+- [ ] [Contains a title heading](#contains-a-title-heading)
+- [ ] [Contains brief Objective section](#contains-brief-objective-section)
+- [ ] [Properly formated Procedure section](#properly-formated-procedure-section)
```

Comment:
formated -> formatted and #properly-formated-procedure-section -> #properly-formatted-procedure-section